pkg/apiserver/conprof/service.go

```diff
@@ -111,7 +111,8 @@ func (s *Service) reverseProxy(targetPath string) gin.HandlerFunc {
 				_ = c.Error(rest.ErrBadRequest.WrapWithNoMessage(err))
 				return
 			}
-			c.Request.URL.RawQuery = queryStr
+
+			c.Request.URL.RawQuery = strings.Replace(c.Request.URL.RawQuery, "token="+token, queryStr, 1)
```

Comment:
Refine this part logic, after parsing the token , we only replace the token part, and keep other query params.

/cc @shhdgit

Reply:
How about use `URL.Values` to ser/deser the query values.

Reply:
I found a way that doesn't need to change this part logic, so I reverted the changes.